0.download_data/0.download_training_movies.sh

```diff
@@ -0,0 +1,2 @@
+jupyter nbconvert --to python aspera_training_downloader.ipynb
+python aspera_training_downloader.py
```

Comment:
```suggestion
python aspera_training_downloader.py

```

To remove that red symbol at the end of this file (add a new line character)